config/configmanager/config.go

```diff
@@ -50,9 +50,36 @@ func LoadFrom(file string) (config.Config, error) {
 	if err != nil {
 		return c, fmt.Errorf("could not load config from file: %w", err)
 	}
+
+	err = ValidateConfig(c)
+	if err != nil {
+		return c, err
+	}
 	return c, nil
 }
 
+// ValidateConfig validates config before we use it
+func ValidateConfig(c config.Config) error {
+
+	// cpuType validation dependent on qemu; harder to do here
+
+	validnetworkDrivers := map[string]bool{"gvproxy": true, "slirp": true}
+	if _, ok := validnetworkDrivers[c.Network.Driver]; !ok {
+		return fmt.Errorf("invalid networkDriver: '%s'", c.Network.Driver)
+	}
+
+	validMountTypes := map[string]bool{"9p": true, "sshfs": true, "virtiofs": true}
+	if _, ok := validMountTypes[c.MountType]; !ok {
+		return fmt.Errorf("invalid mountType: '%s'", c.MountType)
+	}
+	validVMTypes := map[string]bool{"qemu": true, "vz": true}
+	if _, ok := validVMTypes[c.VMType]; !ok {
+		return fmt.Errorf("invalid vmType: '%s'", c.VMType)
```

Comment:
To fix the integration tests error, this should be under the condition of `if util.MacOS13orNewer`

Reply:
Done, thanks